ql/ql/test/callgraph/ParamModules.qll

```diff
@@ -0,0 +1,63 @@
+module PredicateSig {
+  signature predicate fooSig(int i);
+
+  module UsesFoo<fooSig/1 fooImpl> {
+    predicate bar(int i) { fooImpl(i + 1) }
+  }
+
+  predicate myFoo(int i) { i = 42 }
+
+  predicate use(int i) { UsesFoo<myFoo/1>::bar(i) }
```

Comment:
I may be mistaken, but I don't see anything that `UsesFoo` resolves to here. Is this intended? Would it perhaps make sense to resolve it to `module UsesFoo` above?

Reply:
I don't get your comment, the `UsesFoo` here is resolved to the `module UsesFoo` above.

Reply:
As far as I can tell from the expected output, we resolve `bar` to the `bar` on line 5, and `myFoo` to the `myFoo` on line eight, but I would also expect `UsesFoo` to resolve to the `UsesFoo` on line 4. Where is this evident in the `.expected` output?

(Actually, I guess I have my answer: this is only a test of the call graph, and `UsesFoo` isn't being called. I'll assume, then, that it resolves as you say.)